homeassistant/components/linear_garage_door/diagnostics.py

```diff
@@ -0,0 +1,26 @@
+"""Diagnostics support for Linear Garage Door."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import LinearUpdateCoordinator
+
+TO_REDACT = {CONF_PASSWORD, CONF_EMAIL}
```

Comment:
Move to const.py

Reply:
Why move to `const.py`? It's only going to be used in this file anyway